managers/opportunity/index.js

```diff
@@ -39,20 +39,134 @@ class opportunityManager {
     }
   }
 
+  // http://localhost:3030/opportunity?limit=4&page=2, Example of how to use the Get Method
+
+  //  page => Means the Current Page that the User Wants
+  //  limit => The count of pages that should be rendered starting from the given page
```

Comment:
limit => The count of pages that should be rendered starting from the given page

limit => The count of opportunities that should be rendered starting from the first opportunity on given page